esp/xslt/esdl2monitor.xslt

```diff
@@ -154,172 +142,135 @@ MonSoapcall(string requestXml) := FUNCTION
-
-  string monAction := 'Create' : STORED('MonAction', FORMAT(SELECT('Create,Run,DebugCreate,DebugInsert,DebugCompare'), SEQUENCE(4)));
-  string cassandraServer := '127.0.0.1' : stored('cassandraServer', FORMAT(SEQUENCE(1)));
-  string queryUrl := '' : stored('URL', FORMAT(SEQUENCE(3)));
-  string MonitorIdIn := '' : stored('MonitorId', FORMAT(SEQUENCE(5)));
-  string userid := 'xyz123' : stored('UserId', FORMAT(SEQUENCE(6)));
-
-
 // Initialize the Cassandra table, passing in the ECL dataset to provide the rows
 // When not using batch mode, maxFutures controls how many simultaenous writes to Cassandra are allowed before
 // we start to throttle, and maxRetries controls how many times inserts that fail because Cassandra is too busy
 // will be retried.
 
-addMonitor(dataset(monitorStore) values) := EMBED(cassandra : server(cassandraServer), user('eclmon'), keyspace('eclresultmonitoring'), maxFutures(100), maxRetries(10))
-  INSERT INTO monitors (monitorId, userId, request, result) values (?,?,?,?);
+monitorStoreRec getStoredMonitor(string id) := EMBED(cassandra : server(csndServer), user(csndUser), password(csndPassword), keyspace('eclresultmonitoring'))
+  SELECT monitorId, result from monitors WHERE monitorId=? LIMIT 1;
 ENDEMBED;
 
-monitorStore getStoredMonitor(string id) := EMBED(cassandra : server(cassandraServer), user('eclmon'), keyspace('eclresultmonitoring'))
-  SELECT monitorId, userId, request, result from monitors WHERE monitorId=? LIMIT 1;
-ENDEMBED;
-
-updateMonitor(dataset(monitorStoreUpdate) values) := EMBED(cassandra : server(cassandraServer), user('eclmon'), keyspace('eclresultmonitoring'), maxFutures(100), maxRetries(10))
+updateMonitor(dataset(monitorStoreRec) values) := EMBED(cassandra : server(csndServer), user(csndUser), password(csndPassword), keyspace('eclresultmonitoring'), maxFutures(100), maxRetries(10))
   INSERT INTO monitors (monitorId, result) values (?,?);
 ENDEMBED;
 
-
-CreateMonitor (string userid, string url, string requestXml) := MODULE
-  DATA16 monitorHash := HASHMD5(userId, requestXml, TimeLib.CurrentTime(false), '<xsl:value-of select="$queryName"/>');
-  SHARED string monitorId := STD.Str.ToHexPairs(monitorHash);
-  SHARED soapOut := MonSoapCall(requestXml)[1];
-  SHARED responseXML := IF (soapOut.soap_code!=0, ERROR(soapOut.soap_code, soapOut.soap_message), '&lt;Row&gt;' + TOXML(soapOut) + '&lt;/Row&gt;');
-  SHARED mon1 := Dataset([{monitorId, userId, requestXml, responseXML}], monitorStore);
-
-  SHARED MonitorCreateResultRec BuildMonitor(string id, dataset(monitorStore) monitor) :=TRANSFORM
-    SELF.monitor := monitor;
-    SELF.monitorId := id;
-    SELF.diff_report := [];
-    SELF.latest := soapOut;
-  END;
-  EXPORT Result () := FUNCTION
-    RETURN ROW(BuildMonitor(monitorId, mon1));
+  MonitorResultRec := RECORD
+    string id;
+    string responseXML;
+    dataset(the_responseLayout) report;
+//    dataset(the_responseLayout) soap;
```

Comment:
I left these in the generator for now because it allows me to quickly hand edit the generated code to make a debug query.  Don't want to have to regenerate.  Eventually I'll probably remove these entirely.